ambry-replication/src/main/java/com/github/ambry/replication/ReplicaThread.java

```diff
@@ -587,7 +587,7 @@ private ReplicaMetadataResponse getReplicaMetadataResponse(List<RemoteReplicaInf
    * @return List of store keys that are missing from the local store
    * @throws StoreException if store error (usually IOError) occurs when getting missing keys.
    */
-  private Set<StoreKey> getMissingStoreKeys(ReplicaMetadataResponseInfo replicaMetadataResponseInfo,
+  private Set<StoreKey> getRemoteMissingStoreKeys(ReplicaMetadataResponseInfo replicaMetadataResponseInfo,
```

Comment:
Strongly recommend to use getMissingRemoteStoreKeys() or original getMissingStoreKeys().

getRemoteMissingStoreKeys() sounds like get the keys that were missing on remote.

Reply:
I will keep it to original.